client/src/views/client-area/PersonalDataPage.vue

```diff
@@ -1,20 +1,100 @@
 <!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
 
 <template>
-  <div>
-    <h1>PERSONAL DATA</h1>
+  <div class="personal-data-container">
+    <div class="personal-data-content">
+      <ms-summary-card
```

Comment:
Wouldn't if be better to add a `v-if="personalData"` directly on the component? This way it doesn't display at all if those variables are not set, and you could remove the `personalData?.firstname` everywhere.

Reply:
yes